app/config/runtimes-active.php

```diff
@@ -0,0 +1,9 @@
+<?php
```

Comment:
also why do we need a new config? can we just not use a single `runtime` config that has only the active ones?